LORIS_indepth.html

```diff
@@ -516,23 +179,74 @@ <h2>Survey Module</h2>
                       </section>
 
 		    </section>
-		    
-                    <!-- Statistics
-		    <section>
-		      <section>
+              
+    <!-- LORIS key features Column 4 -->
+        <section>
+            
+            <section>
+            <img src="images/LORIS_features.png">
```

Comment:
try to update this image with the 4th row shown the google slides [link here]

Reply:
done